libs/stack/stack-ui/src/components/icons/Error.tsx

```
@@ -0,0 +1,16 @@
+const SvgComponent = (props: React.SVGProps<SVGSVGElement>) => (
+  <svg width="16" height="16" viewBox="16 14 16 16" fill="none" xmlns="http://www.w3.org/2000/svg" {...props}>
+    <g clipPath="url(#clip0_73015_2388)">
+      <path
+        d="M32 22C32 24.1217 31.1571 26.1566 29.6569 27.6569C28.1566 29.1571 26.1217 30 24 30C21.8783 30 19.8434 29.1571 18.3431 27.6569C16.8429 26.1566 16 24.1217 16 22C16 19.8783 16.8429 17.8434 18.3431 16.3431C19.8434 14.8429 21.8783 14 24 14C26.1217 14 28.1566 14.8429 29.6569 16.3431C31.1571 17.8434 32 19.8783 32 22ZM24 18C23.8736 18.0001 23.7486 18.0266 23.6331 18.0779C23.5176 18.1292 23.4141 18.2042 23.3293 18.2979C23.2446 18.3916 23.1803 18.5021 23.1408 18.6222C23.1013 18.7422 23.0874 18.8692 23.1 18.995L23.45 22.502C23.4618 22.6398 23.5248 22.7681 23.6266 22.8616C23.7285 22.9552 23.8617 23.0071 24 23.0071C24.1383 23.0071 24.2715 22.9552 24.3734 22.8616C24.4752 22.7681 24.5382 22.6398 24.55 22.502L24.9 18.995C24.9126 18.8692 24.8987 18.7422 24.8592 18.6222C24.8197 18.5021 24.7554 18.3916 24.6707 18.2979C24.5859 18.2042 24.4824 18.1292 24.3669 18.0779C24.2514 18.0266 24.1264 18.0001 24 18ZM24.002 24C23.7368 24 23.4824 24.1054 23.2949 24.2929C23.1074 24.4804 23.002 24.7348 23.002 25C23.002 25.2652 23.1074 25.5196 23.2949 25.7071C23.4824 25.8946 23.7368 26 24.002 26C24.2672 26 24.5216 25.8946 24.7091 25.7071C24.8966 25.5196 25.002 25.2652 25.002 25C25.002 24.7348 24.8966 24.4804 24.7091 24.2929C24.5216 24.1054 24.2672 24 24.002 24Z"
+        fill="currentColor"
+      />
+    </g>
+  </svg>
+)
+
+const Error = memo(SvgComponent, isEqual)
```

Biome (1.9.4)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)